### DISCUSSION TITLE
第3节课作业(16班)

### DISCUSSION THREAD

**Discussion by Yang-Changhui** [InternLM/Tutorial#167] [作业]
16班第3节课作业在下面进行提交

**Comment by future-2002** [comment]
https://blog.csdn.net/Y2483490891/article/details/135434150?spm=1001.2014.3001.5501

**Comment by Eadral** [comment]
https://www.yuque.com/eadral/ruaugi/gonxuo63plixbbpu?singleDoc

**Comment by Aorg** [comment]
https://github.com/Aorg/Assignment/blob/main/langchain/lesson3_langchain_assignment.md

**Comment by Aitejiu** [comment]
https://www.cnblogs.com/aitejiu/p/17956111

**Comment by Star-cre** [comment]
https://zhilengnuan.blog.csdn.net/article/details/134491154?spm=1001.2014.3001.5502

**Comment by 2404589803** [comment]
https://2404589803.github.io/My-datawhale-learning/#/shusheng_puyu_learnning53/ch03

**Comment by Durian-1111** [comment]
**Day03-Homework**
链接：https://xby48gs45u.feishu.cn/docx/KQEzdkuiWohvfYx6YjNcV3IFnrf?from=from_copylink

**Comment by Dhaizei** [comment]
http://t.csdnimg.cn/62NHe

**Comment by xuanxiaochenmi** [comment]
https://github.com/xuanxiaochenmi/InternLLM_notes/blob/master/lesson_3/lesson_3.md

**Comment by Bob-Y** [comment]
基础作业：复现课程知识库助手搭建过程 (截图)
<img width="960" alt="微信图片_20240111090934" src="https://github.com/InternLM/tutorial/assets/31715275/71bfcf51-12c4-46e0-8f90-90d5a2c0a091">
<img width="960" alt="微信图片_20240111090940" src="https://github.com/InternLM/tutorial/assets/31715275/9989414c-2619-40d9-ab20-263ff90d5723">

**Comment by morcake** [comment]
https://blog.51cto.com/morcake/9186343

**Comment by EnableAsync** [comment]
https://enableasync.github.io/internlm/internlm-03/

**Comment by hhhanqian** [comment]
http://xhslink.com/HKaTcz

**Comment by etheral12138** [comment]
https://y1yc0dwhb0o.feishu.cn/docx/GFqZdBAFMol7uKxqO9xcEEMFnm3?from=from_copylink

**Comment by zszbyzsz** [comment]
https://juejin.cn/post/7322275119594176575

**Comment by TianCai19** [comment]
https://codetwice.notion.site/aad22bc70c6944bcb7331a4ccc162460?pvs=4
